charts/base-cluster/values.schema.json

```diff
@@ -1263,6 +1263,102 @@
       },
       "additionalProperties": false
     },
+    "weaveworks": {
+      "type": "object",
+      "description": "Values for the gitops-chart",
+      "properties": {
+        "enabled": {
+          "type": "boolean"
+        },
+        "adminUser": {
+          "type": "object",
+          "properties": {
+            "create": {
+              "type": "boolean"
+            },
+            "username": {
+              "type": "string"
+            },
+            "createSecret": {
+              "type": "boolean"
+            },
+            "createClusterRole": {
+              "type": "boolean"
+            },
+            "passwordHash": {
+              "type": "string"
+            }
+          }
+        },
+        "ingress": {
+          "type": "object",
+          "properties": {
+            "enabled": {
+              "type": "boolean"
+            },
+            "host": {
+              "type": "string"
+            },
+            "tls": {
+              "type": "array"
+            },
+            "className": {
+              "type": "string"
+            }
+          }
+        },
+        "oidcSecret": {
+          "type": "object",
+          "properties": {
+            "additionalKeys": {
+              "type": "object"
+            },
+            "create": {
+              "type": "boolean"
+            }
+          }
+        },
+        "replicaCount": {
+          "type": "integer"
+        },
+        "rbac": {
+          "additionalRules": {
+            "type": "array"
+          },
+          "create": {
+            "type": "boolean"
+          },
+          "impersonationResourceNames": {
+            "type": "array"
+          },
+          "impersonationResources": {
+            "type": "array"
+          },
+          "viewSecretsResourceNames": {
+            "type": "array"
+          }
+        },
```

Comment:
What is this used for? We should provide a common option or just leave this out

I'm in favor of the latter

Reply:
than lets drop it. The default values should to the job perfectly fine :+1: